Resources/Private/JavaScript/asset-collections/src/components/DeleteButton.tsx

```diff
@@ -28,9 +28,11 @@ const DeleteButton: React.FC = () => {
             if (!canDeleteTag) return;
             // TODO: Implement `obtainApprovalToDeleteCollection` for deleting
             const confirm = window.confirm(
-                translate('action.deleteTag.confirm', 'Do you really want to delete the tag ' + selectedTag.label, [
-                    selectedTag.label,
-                ])
+                translate(
+                    'action.deleteTag.confirm',
+                    'Do you really want to delete the tag ' + selectedTag.label + '?',
```

Comment:
We should use the template string here and wrap the tag name in quotes to avoid confusion with some tag names.

Reply:
Done - hopefully as you expected.